streamerate/streams.py

```diff
@@ -590,10 +592,81 @@ def flatMap(self, predicate: Callable[[_K], Iterable[_V]] = _IDENTITY_FUNC) -> "
             return stream(ItrFromFunc(lambda: itertools.chain.from_iterable(self)))
         return stream(ItrFromFunc(lambda: itertools.chain.from_iterable(self.map(predicate))))
 
+    def makeMapping(
```

Comment:
`makeMapping`: consider using `pairWith` to highlight that each element is paired with the result of applying `f`